README.md

```diff
@@ -13,6 +13,21 @@ To automate the administration of `greenelab/scrum` issues, this repository reli
 Every day, Travis CI executes the commands in [`.travis.yml`](.travis.yml).
 As appropriate, **@scrum-lord** closes and opens issues to keep the scrum issues up to date.
 
+## Deployment Instructions
+
+Use these instructions to deploy a new instance of the scrumlord to manage scrum issues for a repository
+
+1. Fork repo
+2. Create new empty repo organization/scrum
+3. Get Github login token (https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) with repo scope
+   - The safest way to do this is to create a new machine user that doesn't have any other privileges than for the scrum repo
+4. log into https://travis-ci.org using github
```

Comment:
New CI setups should use https://travis-ci.com. See https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps